pygeos/test/test_linear.py

```diff
@@ -154,3 +154,23 @@ def test_shared_paths_non_linestring():
     g2 = pygeos.points(0, 1)
     with pytest.raises(pygeos.GEOSException):
         pygeos.shared_paths(g1, g2)
+
+
+def test_shortest_line():
```

Comment:
Can you also test with a prepared geometry?

Reply:
Ah, good point, I forgot to test with prepared geometries